src/main/java/core/basesyntax/Calculator.java

```
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+public class Calculator {
+    public double calculate(double a, double b, char operation) {
+        //double result = 0;
+        switch (operation) {
+            case '+':
+                checkForDoubleMinValue(a, b);
+                checkForDoubleMaxValue(a, b);
```

Why do you check these values? They both are valid double numbers if you add or subtract them you will get Infinity that is also a valid double value

---

I have made tests so that the method should throw an exception in case of DoubleMinValue or ForDoubleMaxValue